.github/workflows/docs.yml

```
@@ -8,16 +8,13 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: macOS-latest
```

Why do this?

macOS machines are considerably more expensive so feels like unnecessary abuse of public CI infrastructure... Not to mention it's a lot easier to spin up a Linux VM/Docker container to reproduce errors than run a macOS VM.

```suggestion
    runs-on: ubuntu-latest
```